app/views/layouts/viewer_layout.html.erb

```diff
@@ -2,6 +2,7 @@
 <head>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <%= csrf_meta_tag %>
+    <script src="https://cdn.jsdelivr.net/npm/hls.js@1"></script>
```

Comment:
Does it not work without this? If the mime type is the fix maybe bringing in hls via vite is sufficient?